gatsby-node.js

```diff
@@ -39,6 +39,8 @@ exports.createSchemaCustomization = ({ actions }) => {
         slug: String
         path: String
         fullPath: String
+				is_keynote: Boolean
+				is_sponsor: Boolean
```

Comment:
Tabs?? 😱 😆 

```suggestion
        is_keynote: Boolean
        is_sponsor: Boolean
```

Reply:
The horror 😱 (fixed lol)

Reply:
And prettier too 😅 

![](https://media2.giphy.com/media/WTX421Dn6TexGtdK8R/giphy.gif?cid=ecf05e47pqfz3s172mhplify02cjukujtgffbnhxy1lwcrtr&ep=v1_gifs_search&rid=giphy.gif&ct=g)